### ISSUE TITLE
Feature request: Add lexicon support for element descriptions

### ISSUE THREAD

**Issue by sdrenth** [modxcms/revolution#13614]
### Summary
Would be nice if we could use lexicons for element descriptions, think of:

- TV title
- TV description
- TV options (for select list)
- Template names
- Template description

etc.

**Comment by jonleverrier** [comment]
It's been discussed here:
https://github.com/modxcms/revolution/pull/11394

But not sure what the final discussion concluded.

**Comment by Oetzie** [comment]
I'm going to try and fix this today #modxbughunt

**Comment by OptimusCrime** [comment]
Any luck @Oetzie ?

**Comment by Oetzie** [comment]
A status update for this feature.

- Categories supports lexicons:
    - In the MODX trees
    - In the MODX combo's (getlist processor)
    - In the resource TV panel
- TVs supports lexicons:
    - In the MODX trees
    - In the resource TV panel
    - In the 'TV tab' of a template
- Templates supports lexicons:
    - In the MODX trees
    - In the MODX combo's (getlist processor)

What I still need to fix is that TV options can support lexicons, after that I will create a pull request.

I made a new namespace called 'Lexicon' and that namespace holds all the translations. I did not make any language files in the core/lexicon/ directory, but I made the language files in the core/components/lexicon directory so the translations won't be overwritten in an update.

**Comment by azeller** [comment]
Big +1 for this issue.

**Comment by hugopeek** [comment]
Yes, this is important. I feel a bit  embarrassed sometimes when promoting MODX as multilingual, while not being able to give editors instructions in their own language above a TV. And for an Arabic project, this proved especially cumbersome.

**Comment by ghost** [comment]
Something along the lines of ModDevTools might work well for this, its code might be worth looking at.

**Comment by Ibochkarev** [comment]
@Oetzie Hello! Did it turn out to complete the decision on this request?
